packages/table/src/TableHeaderCell.tsx

```diff
@@ -19,8 +20,10 @@ type Props<T extends IdType> = {
   scrollable?: boolean;
   /** This determines whether the table is sortable or not. **/
   sortable?: boolean;
-  /**  **/
+  /** Correlates with whether the sorting is done outside of the table component or not. If sorting is performed outside of the component, this should be set to true. **/
   manualSortBy?: boolean;
+  /** When true, it will take the width as defined in the column configuration and apply it to the styles of each column.  */
```

Comment:
```suggestion
  /** Correlates with whether the sorting is done outside of the table component or not. If sorting is performed outside of the component, this should be set to true. */
  manualSortBy?: boolean;
  /** When true, it will take the width as defined in the column configuration and apply it to the styles of each column. */
```